workflow-steps/cache/output/post.js

```diff
@@ -525,12 +525,12 @@ var require_minimatch = __commonJS({
   "../../node_modules/minimatch/minimatch.js"(exports, module2) {
     module2.exports = minimatch;
     minimatch.Minimatch = Minimatch;
-    var path = function() {
+    var path = (() => {
```

Comment:
interesting the post file had modifications even though you didn't make changes - maybe it's the different node version?

Reply:
I was wondering the same thing. I have 20.9.0 fwiw